src/internal/js/mod.rs

```diff
@@ -307,10 +313,26 @@ impl JsString {
         }
     }
 
+    pub fn ucs2_value(self) -> Vec<u16> {
```

Comment:
I think it might be worth to expose slice directly from pointer & length provided by V8 - that way you need only one method instead of two, and can avoid unnecessary allocations if you just need to read the data.